.github/workflows/pr.yml

```diff
@@ -290,6 +290,125 @@ jobs:
           if [ "$numPass" -lt 36 ];then echo "*** Failed to pass at least 36 ! ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** Passed! ***";exit 0;fi
 
+  functional_docker_rootless_containerd_ubuntu:
```

Comment:
At the very bottom of the file in the `upload all reports` section just need to copy the reports from this job to `all_reports`

Reply:
done